lm75bd/lm75bd.c

```diff
@@ -27,7 +27,17 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
+  uint8_t sendBuf[1] = {0x0U};
+  uint8_t buffer[2];
+  i2cSendTo (devAddr, sendBuf, 1U);
+  i2cReceiveFrom (devAddr, buffer, 2U);
```

Comment:
- [ ] wrap both lines inside of a RETURN_IF_ERROR_CODE 
- [ ] use sizeof(sendBuf) instead of 1
- [ ] use sizeof(buffer) instead of 2